src/dev/tslint/rules/moduleMigrationRule.js

```diff
@@ -17,21 +17,37 @@
  * under the License.
  */
 
+const path = require('path');
 const Lint = require('tslint');
 
+const KIBANA_ROOT = path.dirname(path.resolve(__dirname, '../../..'));
```

Comment:
Another option would be to use `REPO_ROOT` from `../../constants`

Reply:
I like that better, though would need to convert the rule to Typescript which I would like to avoid since all these rules will soon go away when we migrate to eslint.

Reply:
Ah, right, forgot that this code is loaded directly by tslint